Sources/DolbyIORTSUIKit/Private/Views/ErrorView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ErrorView.swift
+//
+
+import DolbyIOUIKit
+import SwiftUI
+
+struct ErrorView: View {
+    private var theme: Theme = ThemeManager.shared.theme
```

Comment:
Is this `theme` used anywhere here?

Reply:
You are right @CasperMok - it is not. I have now removed that line of code.